### ISSUE TITLE
502 Bad Gateway from russellsstewart.com 

### ISSUE THREAD

**Issue by canico** [TensorBox/TensorBox#236]
the script download_data.sh can not download the pre-trained models from russellsstewart.com, the site seems to be down.

```bash
ubuntu@xxxxxx:~/tensorbox$ bash download_data.sh 
Downloading...
--2017-04-26 15:15:52--  http://russellsstewart.com/s/tensorbox/inception_v1.ckpt
Resolving russellsstewart.com (russellsstewart.com)... 52.8.97.128
Connecting to russellsstewart.com (russellsstewart.com)|52.8.97.128|:80... connected.
HTTP request sent, awaiting response... 502 Bad Gateway
2017-04-26 15:15:52 ERROR 502: Bad Gateway.
```

**Comment by goofcode** [comment]
It's fixed now!

**Comment by canico** [comment]
Thanks, I'll try it right now!

**Comment by LuxunHuang** [comment]
Sorry，I try download_data.sh still can not download the pre-trained models.

Downloading...
--2017-05-08 14:32:51--  http://russellsstewart.com/s/tensorbox/inception_v1.ckpt
Resolving russellsstewart.com (russellsstewart.com)... 52.8.97.128
Connecting to russellsstewart.com (russellsstewart.com)|52.8.97.128|:80... connected.
HTTP request sent, awaiting response... 502 Bad Gateway
2017-05-08 14:32:51 ERROR 502: Bad Gateway.

--2017-05-08 14:33:10--  http://russellsstewart.com/s/tensorbox/overfeat_rezoom/save.ckpt-150000v2
Resolving russellsstewart.com (russellsstewart.com)... 52.8.97.128
Connecting to russellsstewart.com (russellsstewart.com)|52.8.97.128|:80... connected.
HTTP request sent, awaiting response... 502 Bad Gateway
2017-05-08 14:33:10 ERROR 502: Bad Gateway.

**Comment by quanghoc** [comment]
Yup I just ran into the same problem (it's been several days). Could you please post this somewhere reliable? Thanks.

**Comment by giuliogatto** [comment]
same here...
Connecting to russellsstewart.com (russellsstewart.com)|52.8.97.128|:80... connected.
HTTP request sent, awaiting response... 502 Bad Gateway

**Comment by HsiangHung** [comment]
it seems I met similar problem. 
During training, tensorflow cannot find `inception_v1.ckpt` and under `data` folder the file doesn't exist. I also recalled during downloading the data, there happened awaiting response problem. Is it the server which hosts the data is down?

**Comment by chok68** [comment]
found one (but not tested it yet)

[https://github.com/bethesirius/ChosunTruck/blob/master/linux/tensorbox/data/inception_v1.ckpt](url)

**Comment by safijari** [comment]
I'm running into this now. Is there another mirror?

**Comment by frnsys** [comment]
can someone that has already downloaded it mirror it please?

**Comment by safijari** [comment]
I have uploaded the ckpt files [here](https://app.box.com/s/pqkum8dq3849chjz4pp9ctjpykrz8s15). The links for the tar.gz files still works and are hosted by Stanford so I think we're safe there. This is from my Box account so if there's lots of activity the link may die. If that happens I will look for other mirrors. I suggest other people affected by this create their own mirrors for the greater good!

**Comment by irwineffect** [comment]
russellsstewart.com is still down.

**Comment by safijari** [comment]
Tell me if my link goes down. As for the maintainer's website, it's anyone's guess
